neoload/commands/test_results.py

```diff
@@ -221,8 +222,14 @@ def create_json(name, description, quality_status, external_url, external_url_la
     if quality_status is not None:
         data['qualityStatus'] = quality_status
     if external_url is not None:
-        data['externalUrl'] = external_url
-    if external_url_label is not None:
+        match = re.match(r'\[(.*)\]\((.*)\)', external_url)
+        if match:
+            data['externalUrl'] = match.group(2)
+            data['externalUrlLabel'] = match.group(1)
+        else:
+            data['externalUrl'] = external_url
+            data['externalUrlLabel'] = ''
```

Comment:
I think we should not reset the externalUrlLabel with empty, but rather use the external_url_label if provided.